### PR TITLE
runtimes/neuron: add support for AWS neuron / trainium / inferentia 2

### PR DESCRIPTION
**Pull request by steeve**
This PR adds support for AWS Neuron / Trainium 1 / Inferentia 2.

It can be enabled via the `--@zml//runtimes:neuron=true` flag.

Everything should work as expected.

### CODE REVIEW

File: runtimes/BUILD.bazel [hunk 1]
```diff
@@ -25,6 +28,39 @@ RUNTIMES = {
     for runtime in RUNTIMES.keys()
 ]
 
+write_file(
```
Comment (gwenzek) [inline runtimes/BUILD.bazel:31]: what's that ?

File: runtimes/BUILD.bazel [hunk 2]
```diff
@@ -25,6 +28,39 @@ RUNTIMES = {
...
+    name = "zml.txt",
+    out = "zml.txt",
+    content = ["ZML loves you <3"],
+)
+
+tar(
```
Comment (gwenzek) [inline runtimes/BUILD.bazel:37]: and this ?

File: runtimes/cpu/BUILD.bazel [hunk 3]
```diff
@@ -25,3 +25,9 @@ zig_library(
         "//conditions:default": [":empty"],
     }),
 )
+
+filegroup(
```
Comment (gwenzek) [inline runtimes/cpu/BUILD.bazel:29]: why ?

**Review by gwenzek — APPROVED** [review]
Noice !

File: runtimes/BUILD.bazel (hunk 2 above)
Comment (steeve) [reply runtimes/BUILD.bazel:37]: needed because else `oci_image` complains that `layers` is empty

**Pull request merged**